usagov/vote-gov-drupal
# Feature/vote 1131/footer px design updates

[VOTE-1131](https://cm-jira.usa.gov/browse/VOTE-1131)

## Description

Implemented the PX design revisions of the footer. This includes both the structure of the footer, placement of components, and the styling.

## Deployment and testing

### Post-deploy steps

1. lando retune
2. Cd into the theme, run: npm run build

### QA/Testing instructions

1. Visit the homepage. Confirm it matches the[ designs in Figma](https://www.figma.com/design/u82ICmmHNBKpE1Tw4nsO0K/Vote.gov-Redesign-2024-(Master)?node-id=8012-20081&m=dev) for all the default breakpoints (left column).
2. Confirm design matches for all the languages and that there are no regressions.

## Checklist for the Developer




## Checklist for the Peer Reviewers
